addons/nightvision/stringtable.xml

```diff
@@ -283,37 +283,62 @@
             <German>Visierunschärfe</German>
             <Chinese>瞄準具模糊程度</Chinese>
             <Chinesesimp>瞄准具模糊程度</Chinesesimp>
-            <Italian>Blur durante la mira</Italian>
```

Comment:
Why are you removing italian translations?

Reply:
It wasn't intentional. I'm new to github, when I started work on this PR I cloned ACE instead of making a fork. The Italian translation was added between when I started and when I learned how to fork, and I used the out of date stringtable without updating it by mistake.

Italian translation should be in with the last commit.

I still need to make sure UAVs are handled correctly with commy2's fix for the CBA issue, so this still needs work.